dashboard/app/controllers/code_reviews_controller.rb

```diff
@@ -46,7 +46,7 @@ def create
     authorize! :create, code_review, project_owner_id
     code_review.save!
 
-    render json: code_review.summarize
+    render json: code_review.summarize_with_comments
```

Comment:
Why this change and the one below?

Reply:
I use the response to set to the state of the component. It would be possible to just modify the state locally for the changes that are made to the record, but this seemed more straight-forward